app/src/main/res/values/strings.xml

```diff
@@ -468,6 +468,8 @@
     <string name="wallpaper_tap_to_change_switch_label">Tap logo to change wallpaper</string>
     <!-- Description for switch which toggles the "tap-to-switch" behavior on home screen logo -->
     <string name="wallpaper_tap_to_change_switch_description">Cycle through and update image without leaving the homepage.</string>
+    <!-- Content description for switch which toggles the "tap-to-switch" behavior on home screen logo -->
+    <string name="wallpaper_content_description_logo">Firefox logo - change the wallpaper, button</string>
```

Comment:
for my edification, does UX provide content descriptions or do we kinda come up with them ourselves?

Reply:
Sometimes they do and some times they provide them 😄 . Normally if it's something straightforward I do it myself otherwise, I'll ask UX :)